src/tests/proctests.cc

```diff
@@ -458,6 +458,64 @@ void test_proc_term_restart4()
     sset.remove_service(&p);
 }
 
+// ToDo: There is some problems around exit_status handling in test environment
```

Comment:
Change ToDo to TODO, stick with the same convention already used elsewhere.
Grammar: There _are_ some problems ...

Reply:
(But more accurately: There _is a problem_ ...)